front/app/utils/scroll.ts

```diff
@@ -25,3 +27,17 @@ export function scrollToElement({
     element.focus();
   }
 }
+
+export const scrollToTop = () => {
+  // Cannot use hooks
+  const isMobileOrSmaller = window.innerWidth <= viewportWidths.tablet;
+
+  // On some mobile browsers, we need a timeout for this to work correctly
+  if (isMobileOrSmaller) {
+    setTimeout(() => {
+      window.scrollTo(0, 0);
+    }, 100);
```

Comment:
I tested on my physical device (on an epic deployment), and indeed this timeout was necessary. Since this is a function I can't use our `useBreakpoint` hook, but I hope using the `viewportWidths` directly is fine?

Reply:
I think it's not so bad, as long as we use the constants from the component library 👍